gitclone/gitclone_test.go

```diff
@@ -152,10 +153,7 @@ func Test_checkoutState(t *testing.T) {
 				PRMergeBranch: "pr_test",
 				ShouldMergePR: true,
 			},
-			wantCmds: []string{
-				`git "fetch" "--jobs=10" "--no-tags" "--no-recurse-submodules" "origin" "refs/pr_test:refs/remotes/pr_test"`,
-				`git "checkout" "refs/remotes/pr_test"`,
-			},
+			wantErrType: ParameterValidationError{},
```

Comment:
This test case is a little weird, I don't think it's valid that a build is triggered with a merge ref but without a head ref, so I changed the test to expect an error.